packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -513,7 +513,7 @@ describe("asset-server handler", () => {
 		}
 
 		expect(earlyHintsRes.headers.get("link")).toMatchInlineSnapshot(
-			`"</a.png>; rel=\\"preload\\"; as=image, </b.png>; rel=\\"preload\\"; as=image"`
+			`"</a.png>; rel="preload"; as=image, </b.png>; rel="preload"; as=image"`
```

Comment:
Not sure why this had the escape characters before and not now, but it appears to match what early hints should look like: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/preload

Reply:
This happened after rebase, so I'm presuming it's due to changes in miniflare / etc and may not reflect prod. Hopefully we can move these tests to the Vitest pool soon.